templates/dashboard/base_modal.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+
+<form
+    role="form" method="post" action="{% block action_url %}#!{% endblock %}"
+    class="{% block form_class %}form-async{% endblock %}" novalidate>
+    {% csrf_token %}
+    <div class="modal-content">
+        <div class="row">
+            <div class="col s12">
+                <h5>
+                    {% block title %}{% endblock %}
+                </h5>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col s12">
+                {% block content %}{% endblock %}
+            </div>
+        </div>
+    </div>
+    <div class="modal-footer">
+        <a href="#!" class="modal-action modal-close btn-flat">
+            {% block cancel_action %} {% trans "Cancel" context "Dashboard modal cancel action" %} {% endblock %}
+        </a>
+        <button type="submit" class="modal-action btn-flat">
+            {% block primary_action %} {% trans "Submit" context "Dashboard modal primary action" %} {% endblock %}
```

Comment:
I'm not a fan of this. Do we ever want to display a generic "Submit" button (or "Wyślij" when localized)?

Reply:
I suppose we don't. I could omit this text and leave an empty block to force passing text value for primary action.